cumulus/test/runtime/src/lib.rs

```diff
@@ -83,8 +88,23 @@ impl_opaque_keys! {
 /// The para-id used in this runtime.
 pub const PARACHAIN_ID: u32 = 100;
 
-const UNINCLUDED_SEGMENT_CAPACITY: u32 = 3;
+#[cfg(not(feature = "elastic-scaling"))]
+const UNINCLUDED_SEGMENT_CAPACITY: u32 = 4;
+#[cfg(not(feature = "elastic-scaling"))]
 const BLOCK_PROCESSING_VELOCITY: u32 = 1;
+
+#[cfg(feature = "elastic-scaling")]
+const UNINCLUDED_SEGMENT_CAPACITY: u32 = 6;
```

Comment:
we used 7 for this in tests. did we find the root-cause of this? was it fixed?

Reply:
IIRC we used 7 here together with the slot drift to fix the block spikes every 8 blocks. Also we found that max_candidate_depth=6 equals an unincluded segment of 7.